linkis-engineconn-plugins/engineconn-plugins/jdbc/src/main/scala/org/apache/linkis/manager/engineplugin/jdbc/executer/JDBCEngineConnExecutor.scala

```diff
@@ -71,10 +72,18 @@ class JDBCEngineConnExecutor(override val outputPrintLimit: Int, val id: Int) ex
 
     if (properties.get(JDBCEngineConnConstant.JDBC_URL) == null) {
       info(s"The jdbc url is empty, adding now...")
-      val globalConfig = Utils.tryAndWarn(JDBCEngineConfig.getCacheMap(engineExecutorContext.getLabels))
+      val globalConfig: util.Map[String, String] = Utils.tryAndWarn(JDBCEngineConfig.getCacheMap(engineExecutorContext.getLabels))
+      var dataSourceInfo: util.Map[String, String] = new util.HashMap[String, String]()
       if (StringUtils.isNotBlank(dataSourceName)) {
         info("Start getting data source connection parameters from the data source hub.")
-        // todo get data source info by data source client
+        Utils.tryCatch {
+          dataSourceInfo = JDBCMultiDatasourceParser.queryDatasourceInfoByName(dataSourceName, execSqlUser)
+        } {
+          e: Throwable => return ErrorExecuteResponse(e.getMessage, e)
```

Comment:
It is suggested to modify the message to “Failed to get datasource info from datasource server”